db/version_set.h

```diff
@@ -747,6 +752,8 @@ struct ObsoleteFileInfo {
 
 class BaseReferencedVersionBuilder;
 
+// VersionSet is the collection of versions of all the column families of the
+// database. Each database owns one VersionSet.
```

Comment:
This is good, but can you also explain how column family is related to this data structure?

Reply:
Sure, will update.